cue/model/api/v1/secret/oauth_go_gen.cue

```diff
@@ -0,0 +1,22 @@
+// Code generated by cue get go. DO NOT EDIT.
+
+//cue:generate cue get go github.com/perses/perses/pkg/model/api/v1/secret
+
+package secret
+
+import (
+	"net/url"
+	"golang.org/x/oauth2"
+)
+
+#PublicOAuth: {
+	clientID:         #Hidden @go(ClientID)
+	clientSecret:     #Hidden @go(ClientSecret)
+	clientSecretFile: string  @go(ClientSecretFile)
+	tokenURL:         string  @go(TokenURL)
+	scopes: [...string] @go(Scopes,[]string)
+	endpointParams: url.#Values       @go(EndpointParams)
+	authStyle:      oauth2.#AuthStyle @go(AuthStyle)
+}
+
+#OAuth: _
```

Comment:
@AntoineThebaud we should have the new OAuth struct here. I don't remember what you told me to get it or if there was a solution. Can you help us for this issue please ?

Reply:
ah yes indeed. We missed that. We will have to create patch files like `authorization_patch.cue`, but that's not under the scope of this PR.

So it's good here.